docs/admin/program_modules.rst

```diff
@@ -639,6 +649,16 @@ Admin Admissions Dashboard
 Provides an interface for admins to review all of the applications in the
 program. For more information, see `</docs/admin/student_apps.rst>`_.
 
+Lottery Student Registration Big Board
+--------------------------------------
+
+Provides a page for watching the current number of student registrations.
+You can get to it from the link "Student Registration Big Board" on the main
+program management page, or at /manage/[program]/[instance]/bigboard.  It has
+some of the same statistics as the dashboard, but is a lot faster to load, and
+has some fun extra numbers too.
```

Comment:
Does this have enough data in it that it can be useful for non-lottery registrations? If so, we can remove the "lottery" part of the title. If not, can you make the "lottery" part clear in the description?